src/careamics/models/lvae/noise_models.py

```diff
@@ -307,7 +348,7 @@ def likelihood(self, observations, signals):
         return p + self.tol
 
     def getGaussianParameters(self, signals):
```

Comment:
Update camel casing to snake casing for functions.

Reply:
Let's do it in a separate PR with other formatting. There's too much of that